src/model.test.ts

```diff
@@ -62,6 +62,7 @@ describe('HubApiModel', () => {
       }
     };
     const req = {
+      app: { locals: { arcgisPortal: 'https://devext.arcgis.com' } }, 
```

Comment:
The only thing I would change the consistency is have this arcgis portal point to `https://qaext.arcgis.com`, mainly because the examples that you are using involve a QA site. Seeing the mixture of the QA site name with the `dev` domain is a bit jarring.